packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.tsx

```diff
@@ -257,7 +263,8 @@ function ToolbarButtonsRenderer({ handleApply, handleClose, handleReset }: Filte
   );
 }
 
-function isFilterNonEmpty(rootGroup: PropertyFilterBuilderRuleGroup) {
+/** @internal */
+export function isFilterNonEmpty(rootGroup: PropertyFilterBuilderRuleGroup) {
```

Comment:
I would suggest moving this to `Utils.ts` file to avoid exporting it through barrel file.